README.md

```diff
@@ -62,7 +62,7 @@ djenriquez/vault-ui
 ```
 
 Supported environment variables:
-- `NODE_TLS_REJECT_UNAUTHORIZED` disable TLS server side validation (ex. vault deployed with self-signed certificate)
+- `NODE_TLS_REJECT_UNAUTHORIZED` 0 or 1 (default: 0), disable TLS server side validation (ex. vault deployed with self-signed certificate)
```

Comment:
The default confuses me. Typically you would define default as the value held if undefined. If `NODE_TLS_REJECT_UNAUTHORIZED` is undefined, it's equivalent to `NODE_TLS_REJECT_UNAUTHORIZED=1`. Did I misunderstand something? If not, please correct and we'll merge in.